Source/WebCore/dom/ViewTransition.cpp

```diff
@@ -491,8 +490,10 @@ void ViewTransition::activateViewTransition()
         return;
 
     // FIXME: Set rendering suppression for view transitions to false.
-
-    // FIXME: If transition’s initial snapshot containing block size is not equal to the snapshot containing block size, then skip the view transition for transition, and return.
+    if (protectedDocument()->renderView() && protectedDocument()->renderView()->sizeForCSSLargeViewportUnits() != m_initialLargeViewportSize) {
```

Comment:
We should probably treat the `renderView()` being non-existent as a size change, if it's even possible.